src/core/interp.c

```diff
@@ -5632,6 +5633,11 @@ void MVM_interp_run(MVMThreadContext *tc, void (*initial_invoke)(MVMThreadContex
                 cur_op += 20;
                 goto NEXT;
             }
+            OP(cpucores): {
```

Comment:
It's fine if people wish to squash/rebase/tidy their commits before they're merged, but yeah, we don't insist on it. :-)